src/components/Table/TableToolbarComponent.jsx

```diff
@@ -53,40 +47,44 @@ class TableToolbarComponent extends Component {
 
     this.filterTimeoutId = setTimeout(() => {
       this.filterData();
-    }, 2000);
+    }, 500);
 
   };
 
   buildSelectOptions = () => {
     const {filterProperties, properties} = this.props;
 
     return filterProperties.map((item, index) => {
-      return <MenuItem key={index} value={item}
-        primaryText={properties[item].title}
-      />;
+      return <option key={index} value={item}>{properties[item].title}</option>;
     });
   };
 
   render() {
     return (
-      <Toolbar>
-        <ToolbarGroup>
-          <RaisedButton onClick={this.props.handleOpenModal}
-            label={'Add new ' + this.props.singular}
-            style={raisedButtonStyle}
-            primary={true}
-          />
-        </ToolbarGroup>
-        <ToolbarGroup>
-          <SelectField floatingLabelText={'Filter by'} value={this.state.filterProperty}
-            onChange={this.handleMenuItemSelected}>
-            {this.buildSelectOptions()}
-          </SelectField>
-          <TextField floatingLabelText={'Search'} value={this.state.filterValue}
-            onChange={this.handleSearchChange}
-          />
-        </ToolbarGroup>
-      </Toolbar>
+      <div>
+        <div className="pt-navbar-group pt-align-left">
+          <button type="button" className="pt-button pt-intent-primary pt-icon-add"
+            onClick={this.props.handleOpenModal}>{'New ' + this.props.singular}
+          </button>
```

Comment:
I recommend using `Button` component: http://blueprintjs.com/docs/#components.button
I am using it in auth components - so there will be more coherent.